app/src/main/java/vn/mbm/phimp/me/image/CropImage.java

```diff
@@ -354,6 +354,105 @@ public void onClick(View v)
 			/*
 			 * Danh - Add event for button Sepia image effect - End
 			 */
+
+            findViewById(R.id.btnalpha).setOnClickListener(
+                    new View.OnClickListener() {
```

Comment:
Remember that you have used this same code with some modifications last time, it would be better to have them inside a class so that you could call them, for example you have: imageFilters.class with all the filter functions like imageToBlur
Then you have to have XML button clicks or add an on click event listener to execute that function. This will enhance reusablitity of code